airflow/providers/google/cloud/hooks/dataprep.py

```diff
@@ -29,6 +29,20 @@
 from airflow.hooks.base import BaseHook
 
 
+def _get_field(extras: dict, field_name: str):
+    """Get field from extra, first checking short name, then for backcompat we check for prefixed name."""
+    backcompat_prefix = "extra__dataprep__"
+    if field_name.startswith("extra_"):
```

Comment:
In some of these PRs you are using "extra_", some use "extra__" (two underscores), and others are using `backcompat_prefix`.

Reply:
No reason just someone suggested adding double underscore and I simply didn't get to all of them... I can do a cleanup pr after all the dust is settled